src/analysis/normed_space/lp_space.lean

```diff
@@ -1092,3 +1126,175 @@ end
 end topology
 
 end lp
+
+namespace function.injective
+
+variables (E) (𝕜 : Type*) [normed_field 𝕜] [Π i, normed_space 𝕜 (E i)] (p)
```

Comment:
```suggestion
variables {β γ : Type*}
variables (E) (𝕜 : Type*) [normed_field 𝕜] [Π i, normed_space 𝕜 (E i)] (p)
```
and remove the explicit declaration of `β` and `γ` from the lemmas and probably the same for `{φ : β → α} (hφ : injective φ)`

Reply:
I moved the `β γ` to the top of the file